### PR TITLE
Tests for userCard

### LINKED ISSUE #134: Add tests for components

### PR DESCRIPTION
**Pull request by grok-rs**
1) Should redirect to / if user doesn`t exists in sessionStorage
2) Should check links to profilePage are available
3) Should call addFriendToDB by clicking on Add friend
4) Should call createDialog by clicking on Send message
5) Should render correct age from birthday

> How to check Redirect from Routes?

### CODE REVIEW

File: frontend/src/components/shared/userCard/UserCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    render(
+      <MemoryRouter>
+        <UserCard mainInfo={userProps} />
+      </MemoryRouter>
+    );
+  });
+
+  it('Should redirect to / if user doesn`t exists in sessionStorage', () => {
+    expect(getUserFromStorage).toReturnWith(null);
+    //TODO check redirect to login if null
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/userCard/UserCard.test.tsx:69]: is there any code in component that redirects use to login? Cant find anything in userCard

File: frontend/src/components/shared/userCard/UserCard.test.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  });
+
+  it('Should check links to profilePage are available', () => {
+    const wrapper = mount(
+      <MemoryRouter>
+        <UserCard mainInfo={userProps} />
+      </MemoryRouter>
+    );
+    const links = wrapper.find('.leftbar[href="/user/5fae8f5bac677440d079a289"]').length;
+    expect(links).toBeGreaterThan(0);
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/userCard/UserCard.test.tsx]: maybe .toBeEqual(1)?

Comment (oleksandrstarov) [reply frontend/src/components/shared/userCard/UserCard.test.tsx]: also it is better to search for links in wrapper and then check that href prop is equal to required

**Review by oleksandrstarov — APPROVED** [review]

File: frontend/src/components/shared/userCard/UserCard.test.tsx (hunk 1 above)
Comment (grok-rs) [reply frontend/src/components/shared/userCard/UserCard.test.tsx:69]: no, responsible for redirect Routes component, there is middleware, structure 
```
<Routes>
  <Profile>
      <UserCard>
  </Profile>
</Routes> 
```
something like this

**Pull request merged**